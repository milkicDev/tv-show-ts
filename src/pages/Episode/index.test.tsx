import React from 'react';
import renderer from 'react-test-renderer'
import ShowDetails from '.';

it('render app', () => {
  const render = renderer.create(<ShowDetails id={1} />).toJSON();
  expect(render).toMatchSnapshot();
})
