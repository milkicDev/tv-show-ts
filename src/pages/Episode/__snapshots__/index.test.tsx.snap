// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render app 1`] = `
<div>
  <div
    className="header show-header"
  >
    <div
      className="container"
    >
      <div
        className="flex-grid"
      >
        <div
          className="img-placeholder"
        >
          <img />
        </div>
        <div>
          <h3 />
          <div
            className="flex-grid"
            style={
              Object {
                "marginTop": "1rem",
              }
            }
          >
            <span
              className="rating"
              style={
                Object {
                  "direction": "ltr",
                  "display": "inline-block",
                }
              }
            >
              <span
                onClick={[Function]}
                onMouseMove={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                style={
                  Object {
                    "cursor": "inherit",
                    "display": "inline-block",
                    "position": "relative",
                  }
                }
              >
                <span
                  style={Object {}}
                >
                  <span
                    className="fa fa-star-o medium"
                  />
                </span>
                <span
                  style={
                    Object {
                      "display": "inline-block",
                      "left": 0,
                      "overflow": "hidden",
                      "position": "absolute",
                      "top": 0,
                      "width": "0%",
                    }
                  }
                >
                  <span
                    className="fa fa-star medium"
                  />
                </span>
              </span>
              <span
                onClick={[Function]}
                onMouseMove={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                style={
                  Object {
                    "cursor": "inherit",
                    "display": "inline-block",
                    "position": "relative",
                  }
                }
              >
                <span
                  style={Object {}}
                >
                  <span
                    className="fa fa-star-o medium"
                  />
                </span>
                <span
                  style={
                    Object {
                      "display": "inline-block",
                      "left": 0,
                      "overflow": "hidden",
                      "position": "absolute",
                      "top": 0,
                      "width": "0%",
                    }
                  }
                >
                  <span
                    className="fa fa-star medium"
                  />
                </span>
              </span>
              <span
                onClick={[Function]}
                onMouseMove={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                style={
                  Object {
                    "cursor": "inherit",
                    "display": "inline-block",
                    "position": "relative",
                  }
                }
              >
                <span
                  style={Object {}}
                >
                  <span
                    className="fa fa-star-o medium"
                  />
                </span>
                <span
                  style={
                    Object {
                      "display": "inline-block",
                      "left": 0,
                      "overflow": "hidden",
                      "position": "absolute",
                      "top": 0,
                      "width": "0%",
                    }
                  }
                >
                  <span
                    className="fa fa-star medium"
                  />
                </span>
              </span>
              <span
                onClick={[Function]}
                onMouseMove={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                style={
                  Object {
                    "cursor": "inherit",
                    "display": "inline-block",
                    "position": "relative",
                  }
                }
              >
                <span
                  style={Object {}}
                >
                  <span
                    className="fa fa-star-o medium"
                  />
                </span>
                <span
                  style={
                    Object {
                      "display": "inline-block",
                      "left": 0,
                      "overflow": "hidden",
                      "position": "absolute",
                      "top": 0,
                      "width": "0%",
                    }
                  }
                >
                  <span
                    className="fa fa-star medium"
                  />
                </span>
              </span>
              <span
                onClick={[Function]}
                onMouseMove={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                style={
                  Object {
                    "cursor": "inherit",
                    "display": "inline-block",
                    "position": "relative",
                  }
                }
              >
                <span
                  style={Object {}}
                >
                  <span
                    className="fa fa-star-o medium"
                  />
                </span>
                <span
                  style={
                    Object {
                      "display": "inline-block",
                      "left": 0,
                      "overflow": "hidden",
                      "position": "absolute",
                      "top": 0,
                      "width": "0%",
                    }
                  }
                >
                  <span
                    className="fa fa-star medium"
                  />
                </span>
              </span>
            </span>
          </div>
          <div
            className="description"
            dangerouslySetInnerHTML={
              Object {
                "__html": "",
              }
            }
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="container"
  >
    <div
      className="flex-grid space-between"
    >
      <div
        className="w-50"
      >
        <h2>
          show.showInfo
        </h2>
        <div
          className="tableRow"
        >
          <div>
            <label>
              show.streamedOn
            </label>
            <span />
          </div>
          <div>
            <label>
              show.schedule
            </label>
            <span />
          </div>
          <div>
            <label>
              show.status
            </label>
            <span />
          </div>
          <div>
            <label>
              show.genres
            </label>
            <span />
          </div>
        </div>
      </div>
      <div
        className="w-50"
      >
        <h2>
          show.staring
        </h2>
        <div
          className="tableRow"
        />
      </div>
    </div>
  </div>
</div>
`;
